coopr-cli/src/main/java/co/cask/coopr/shell/CLIConfig.java

```diff
@@ -136,16 +146,21 @@ public void setConnection(String host, int port, boolean ssl, String userId, Str
     }
     this.ssl = ssl;
     this.uri = URI.create(String.format("%s://%s:%d", ssl ? "https" : "http", host, port));
-    RestClientManager.Builder builder = RestClientManager.builder(host, port);
+    RestClientConnectionConfig.Builder builder = RestClientConnectionConfig.builder(host, port);
     builder.ssl(ssl);
-    accessToken = getAccessToken(host, port, ssl);
-    builder.accessToken(Suppliers.ofInstance(accessToken));
+
+    try {
+      accessToken = getAccessToken(host, port, ssl);
+      builder.accessToken(Suppliers.ofInstance(accessToken));
+    } catch (IOException e) {
+      // IGNORE
```

Comment:
If there's a connection issue then this will be thrown. The reason for ignoring this was because I didn't want to exit if there's an issue connecting to the default connection, but I've refactored this so that I can throw IOException from here without affect CLI startup.